config/samples/php7_example_loadtest.yaml

```diff
@@ -25,7 +25,7 @@ spec:
     run:
       command:
       - bash
-      - /run_scripts/run_worker.sh
+      - /run_scripts/run_php7_worker.sh
```

Comment:
The new script is called "run_php7_worker", I changed it since I was going to add run_workers.sh to wrap all workers. It turn out to be that I don't have to. I can modify it back to run_worker.sh. :)